### DISCUSSION TITLE
Missing types in [@types/minio] v7.0.10

### DISCUSSION THREAD

**Discussion by marcociarrocca** [DefinitelyTyped/DefinitelyTyped#56199] [Issues with a @types package] [answered]
In @types/minio v7.0.10 the functions:
- getBucketTagging
- setBucketTagging
- removeBucketTagging
- putObjectTagging
- removeObjectTagging
- getObjectTagging
are missing.

[a comment by an automated account is not shown]

**Comment by barinbritva** [comment, marked as answer]
@marcociarrocca, thanks for the issue.
I will add those methods in a couple of days.

**Comment by barinbritva** [comment]
@marcociarrocca, here it is - https://github.com/DefinitelyTyped/DefinitelyTyped/pull/56437

Let me know if you run into some problems
